public/commit_id.txt

```diff
@@ -0,0 +1 @@
+asdf123
```

Comment:
Woops. I just noticed this file. This looks like its not needed. Can we delete this?